src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,41 @@
 
 class Animal {
   // write your code here
+  constructor(health = 100, name) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  };
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor(health, name) {
+    super(100, name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  constructor(health, name) {
+    super(100, name);
+  }
+
+  bite(herbivoreobj) {
```

Comment:
please choose better names -> in example you can see that your argument can be also carnivore, so probably it should be `bite(animal) {` ?